packages/terra-base/tests/nightwatch/NoTranslationsLoadedBase.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Base from '../../lib/Base';
+
+const BaseNoTranslationsLoaded = () => {
+  const noTranslations = (
+    <Base locale="gabcdef" translationsLoadingPlaceholder={<div>No Translations</div>}>
```

Comment:
It should throw not supported locale error message in the console.
```
Uncaught Error: gabcdef is not supported, supported locales:en,en-US,en-GB,es,de,fi-FI,fr,pt
```
That may make the nightwatch test fragile. Could we capture this error message in the spec? I also don't know a better way to test this. Or maybe just remove the nightwatch test.